src/foam/nanos/notification/notifications/NotificationScript.js

```diff
@@ -0,0 +1,12 @@
+foam.CLASS({
+  package: 'foam.nanos.notification.notifications',
+  name: 'NotificationScript',
```

Comment:
I think ScriptRunNotification would be a better name. Then you would have a ScriptRunNotificationNotificationView.